internal/generator/vector/output/loki/loki.go

```diff
@@ -114,6 +114,21 @@ func Conf(o logging.OutputSpec, inputs []string, secret *corev1.Secret, op Optio
 		}
 	}
 	componentID := fmt.Sprintf("%s_%s", id, "remap")
+	if strings.Contains(id, strings.ReplaceAll(fmt.Sprintf("%v", op["syslog"]), "-", "_")) {
```

Comment:
type may help here as well, I don't think we should make decisions based upon this type of string evaluation

Reply:
This is much bigger than it looks like, let's discuss it in a different forum.

Reply:
I forgot to highlight that the check we are doing is against an op property that we set up specifically for this usage in https://github.com/openshift/cluster-logging-operator/pull/2235/files#diff-0b3745488d65f7b49cae4e1d6f675a2250721a4b9fb055584bee8c6250ae4660R50